Docs/roles-and-permissions.md

```diff
@@ -0,0 +1,55 @@
+# Manually setting up roles and assigning permissions in Azure Active Directory (AAD)
+
+## Creating the Application
```

Comment:
The application is created for you when you run a build script, there's no need for a user to create an application explicitly.